From 7289da1a3ada88fabb1227ef344d6aa51b40ffaa Mon Sep 17 00:00:00 2001
From: Mirza Krak <mirza.krak@northern.tech>
Date: Thu, 9 Apr 2020 14:04:47 +0200
Subject: [PATCH 1/1] gstpythonplugin: hard-code libpython path and name

We can override it using --with-libpython-dir=${libdir}, but
it still fails because it looks for a symlinked library ending
in .so instead of the actually library with LIBNAME.so.MAJOR.MINOR

Upstream-status: Inappropriate (Fixed in 1.16 with meson build)

Signed-off-by: Mirza Krak <mirza.krak@northern.tech>
---
 plugin/gstpythonplugin.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/plugin/gstpythonplugin.c b/plugin/gstpythonplugin.c
index b9abafe..4e36afa 100644
--- a/plugin/gstpythonplugin.c
+++ b/plugin/gstpythonplugin.c
@@ -230,7 +230,7 @@ plugin_init (GstPlugin * plugin)
     GST_LOG ("libpython is already loaded");
   } else {
     const gchar *libpython_path =
-        PY_LIB_LOC "/libpython" PYTHON_VERSION PY_ABI_FLAGS "." PY_LIB_SUFFIX;
+        PY_LIB_LOC "/libpython3.7m.so.1.0";
     GST_LOG ("loading libpython from '%s'", libpython_path);
     libpython = g_module_open (libpython_path, 0);
     if (!libpython) {
-- 
2.26.0

